.github/renovate.json5

```diff
@@ -0,0 +1,65 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
+        ":dependencyDashboardApproval"
+    ],
```

Comment:
@HonkingGoose I don't recall ever using `extends` before. The docs at https://docs.renovatebot.com/presets-config/#configrecommended don't say what the explicit values are. Is that intentional? If not, would a good next step be for me to file a docs issue?

Reply:
Hi @JoshuaKGoldberg. Renovate has supported `extends` for 5+ years (inspired by `eslint` terminology). The page you linked to lists which presets it includes:

<img width="733" alt="image" src="https://github.com/eslint/eslint/assets/6311784/63e365cf-84bd-433e-a2fe-9cd9c339f52d">

All the presets listed there are also documented at docs.renovatebot.com and once you drill down you can see the explicit values.

Can you explain where your confusion lies? Maybe I'm not understanding what you mean by "explicit values".

Reply:
> I don't recall ever using `extends` before.

Really? It almost sounds like you usually build your Renovate config from the ground up... 🙃 I sure hope I'm reading this wrong, or you'll have spent a lot of time on basic things...

By the way, we do have docs that try to explain the `extends` config option. [^1]

> All the presets listed there are also documented at docs.renovatebot.com and once you drill down you can see the explicit values.

I think they're trying to say that it's hard to drill down. To drill down the `config:recommended` preset, you'll have to have 7 tabs open for each `extends` item, and then try to keep it all in your head.

> Can you explain where your confusion lies? Maybe I'm not understanding what you mean by "explicit values".

It sounds like they want to see all the fully expanded configuration in a single JSON codeblock. That's probably why they want to file a docs issue.

[^1]: [Renovate docs, `extends` config option](https://docs.renovatebot.com/configuration-options/#extends)

Reply:
> All the presets listed there are also documented at docs.renovatebot.com and once you drill down you can see the explicit values.

> It sounds like they want to see all the fully expanded configuration in a single JSON codeblock. That's probably why they want to file a docs issue.

I'm not confused about anything 😄 @rarkins is right. I'm just being unclear about what I'm trying to say. Sorry. Filed https://github.com/renovatebot/renovate/discussions/24448.

The part that's relevant to this PR is what benefit this specific repo gets from using `config:recommended`. `eslint/eslint` establishes a more fine-grained control of its automations compared to most end-user repositories. Other than `group:babelMonorepo` / `monorepo:babel` I don't see anything relevant to this repo?

Reply:
I think you actually benefit from most of them, for example migrations and workarounds are presets which get updated regularly by the community and there's no reason to think this repo would benefit any less than others.

You certain are welcome/encouraged to break that preset into separate presets though, and exclude any which aren't applicable

Reply:
Yeah, you would probably benefit from `config:recommended`. 😉 

You may use [`ignorePresets`](https://docs.renovatebot.com/configuration-options/#ignorepresets) to ignore parts of `config:recommended` you don't want. For example:

```json
{
  "extends": ["config:recommended"],
  "ignorePresets": ["config:thanks-i-hate-it"]
}
```

Reply:
Thanks - I'll add it back in.